code/controllers/communications.dm

```diff
@@ -137,6 +138,7 @@ var/list/radiochannels = list(
 	"Raider"		= RAID_FREQ,
```

Comment:
the fact that some of these are indented and some of them aren't causes me literal pain